sys/fs/fuse/fuse_vnops.c

```diff
@@ -1360,7 +1359,8 @@ fuse_vnop_link(struct vop_link_args *ap)
 	 * validating that nlink does not overflow.
 	 */
 	if (vap != NULL && vap->va_nlink >= FUSE_LINK_MAX)
-		return EMLINK;
+		return (EXTERROR(EMLINK, "Maximum number of hardlinks "
```

Comment:
Honestly, this error code is so specific that it doesn't even need an extended error message.  Same for the previous EXDEV.